inc/type.class.php

```diff
@@ -1928,6 +1928,13 @@ static function install(Migration $migration) {
 
       $notepad = new Notepad();
       foreach ($allGenericObjectTypes as $genericObjectType => $genericObjectData) {
+         $itemtype = $genericObjectData['itemtype'];
+         if (! class_exists($itemtype, true)) {
+            $warning = "Unable to load the class '$itemtype'\n";
+            $warning .="Please check the directory " . GENERICOBJECT_DOC_DIR . ""; 
```

Comment:
same as above.

The message may be improved; We should find something not too long and boring, and precise enough.